src/main/resources/query/statistics/termTypeFrequency.rq

```diff
@@ -9,13 +9,12 @@ SELECT (?type as ?item) ?label (COUNT(DISTINCT ?term) AS ?count) WHERE {
     GRAPH ?g { ?term a skos:Concept . }
     ?term a-popis-dat-pojem:je-pojmem-ze-slovníku ?vocabulary .
 
+    BIND( ?t as ?t2 )
```

Comment:
This is strange. Is "?t" some kind of input param of the query or why it is here ?

Reply:
It is bound by VALUES later in the StatisticsDao.